woocommerce/woocommerce-android
# Issue/10028 blaze celebration screen

Closes: #10028 


### Description
This PR adds two changes:
- Navigate to the Campaign list after successful campaign creations.
- Show a celebration screen after the first successful campaign creation.

### Testing instructions
1. Sign in to a site using WordPress.com.
2. Open the app, then proceed to campaign creation from any location.
3. Finish the creation.
4. Notice the app opens the campaigns list after the creation, and shows a celebration screen.
5. Repeat 2 and 3, and confirm the celebration screen is not shown anymore.

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/1657201/f5f76509-9e06-433d-93f4-24e8d7b67ec4
